### PR TITLE
Swedish 100%

### PR DESCRIPTION
**Pull request by theseal**
Complete the swedish translation
Finally 100% again!

### CODE REVIEW

File: admin/src/lang.json [hunk 1]
```diff
@@ -458,60 +458,63 @@
...
     "sv": {
-        "saveAt": "Sparad",
-        "restore": "Återskapa",
-        "delete": "Ta bort",
-        "restoreAndDel": "Återskapa & ta bort",
-        "contact": "Kontakt",
-        "settings": "Inställningar",
-        "back": "Tillbaka",
         "about": "Om",
+        "activeTabs": ""Aktiva flikar",
```
Comment (yuanzhoucq) [inline admin/src/lang.json:462]: There is an extra quote mark making this json file invalid.

**Comment by yuanzhoucq** [comment]
Hi @theseal, thanks for your contribution again! Could you please revert the commit for ordering the keys? Because they are grouped by functions. Relative keys have been put together. And there is an extra quote mark making the translation file invalid, which I have pointed out in the in-place comment. 😉

**Comment by theseal** [comment]
Is it better now? Not sure where to add the new keys.

**Comment by yuanzhoucq** [comment]
> Is it better now? Not sure where to add the new keys.

Yes! The English version has the right order. But no worries, just leave them in the end. I will move them if needed. Thanks again for your contribution! 😉

**Pull request merged**